### PR TITLE
Code updates

### PR DESCRIPTION
**Pull request by xcoulon**
- use `r` as the receiver name in the sgmlRenderer
methods
- remove the `prepared` field from the `sgmlRenderer` 
and use it as a package var instead
- remove some aliases
- return `textTemplate` instead of `*textTemplates`

### CODE REVIEW

**Comment by xcoulon** [comment]
@gdamore I'm proposing those changes in your PR so the resulting code is more consistent with what we have today. Please let me know what you think about it.

Also, I noticed that having the templates initialized in the `NewRenderer` func (per this PR) makes the `html5` testing slower, so I suspect that there are cases where the templates are not used. I will need to investigate a little more on that. But for now it's still acceptable.

**Comment by xcoulon** [comment]
> Also, I noticed that having the templates initialized in the `NewRenderer` func (per this PR) makes the `html5` testing slower, so I suspect that there are cases where the templates are not used. I will need to investigate a little more on that. But for now it's still acceptable.

actually, I reintroduced the `sync.Once` thingy but as a package var in f5accb6, and HTML5 test now run fast again ;)

**Comment by gdamore** [comment]
Looks like there are some conflicts... will examine.

**Comment by xcoulon** [comment]
> Looks like there are some conflicts... will examine.

I'm looking at that too, and a couple of other things as well

**Comment by xcoulon** [comment]
> > Looks like there are some conflicts... will examine.
> 
> I'm looking at that too, and a couple of other things as well

I fixed the conflicts, but did squash several commits into a single new one with I pushed with `--force`.

**Comment by xcoulon** [comment]
>     * remove the `prepared` field from the `sgmlRenderer`
>       and use it as a package var instead

I reverted that change in 2101d69 as it makes more sense to have this `sync.Once` as a private field of the `sgmlRenderer` struct rather than at the package level.

**Comment by gdamore** [comment]
Are you at the point where you're ready for me to merge this into my PR?

File: pkg/renderer/sgml/inline_elements.go [hunk 1]
```diff
@@ -15,27 +16,27 @@ type linesRenderer struct {
...
 	}
 	for _, apply := range options {
-		apply(&r)
+		apply(&lr)
 	}
 	// first pass or rendering, using the provided `renderElementFunc`:
 	buf := &bytes.Buffer{}
 	for i, element := range elements {
-		renderedElement, err := r.render(ctx, element)
+		renderedElement, err := lr.render(ctx, element)
```
Comment (gdamore) [inline pkg/renderer/sgml/inline_elements.go:39]: I'd have chosen "le", but I'm fine with this. :-)

**Pull request merged**